Scripts/Source/UD_PurifiedGoo_AME.psc

```diff
@@ -6,9 +6,9 @@ UDCustomDeviceMain Property UDCDmain auto
 Event OnEffectStart(Actor akTarget, Actor akCaster)
     UDCDmain.DisableActor(akTarget)
     int loc_filter = 0xffffffff
-    if Utility.randomInt(0,1) ;50 % chance for suit
-        loc_filter = Math.LogicalAnd(loc_filter,0xFFFFFDFF)
-    endif
+;    if Utility.randomInt(0,1) ;50 % chance for suit
```

Comment:
I think that previous implementation was better, as it allowed the player also see the belt/bra sometimes. Without this, the catsuit will always be equipped.

Reply:
Actually, this random is redundant, thanks to the keywords formlist rearranged, thus ensuring that HeavyBondage it applied first, and suit is after, only if that keyword available. All random calls are already inside RandomManifest.

Reply:
> Please update the PR96_UnforgivingSkyrim.esp . The .esp is using the Cursed Loot textures. This means that the Cursed Loot is now UD master, which should not be the case. Either use different texture or undo the changes, so the Cursed Loot is no master.

Will copying TXTS from Cursed Loot, still pointing to the Cursed Loot files and removed master will be OK?
Or we definitely need to make copies of those files into UD assets?

Reply:
OK, esp has no CursedLott as master anymore, made copies of TXTS and ARMA records, still using files from DCL. Is this OK?
Reverted change in the PurifiedGoo script, filter is set properly.